src/metabase/legacy_mbql/schema.cljc

```diff
@@ -1524,7 +1524,7 @@
    [:skip-results-metadata? {:optional true} :boolean]
    ;; should we skip converting datetime types to ISO-8601 strings with appropriate timezone when post-processing
    ;; results? Used by [[metabase.query-processor.middleware.format-rows]]; default `false`
```

Comment:
is this comment still accurate? `format-rows?` is doing more now, right?

Reply:
Good catch!